### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review
- adds hello-world example for hapi-riot  
  #4

**NOTE:** `<!DOCTYPE html>` tag gets added dynamically but linter complains if it is left out of the `.tag` files

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var Hapi = require('hapi');
+var hoek = require('hoek');
+var Vision = require('vision');
+var HapiRiot = require('../node_modules/hapi-riot/lib/index.js');
```
Comment (nelsonic) [inline lib/index.js]: Well-spotted! issue created: https://github.com/dwyl/hapi-riot/issues/9

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "cover",
     "check-coverage"
   ],
-  "author": "Jack Rans && Jack Carlisle",
+  "author": "@jackcarlisle && @jrans",
```
Comment (nelsonic) [inline package.json:23]: lol. age before beauty? 😆

**Comment by nelsonic** [comment]
@jackcarlisle which linter is complaining about `<!DOCTYPE html>` ?

File: package.json (hunk 2 above)
Comment (jackcarlisle) [reply package.json:23]: 👴🏼👶🏼

**Comment by jackcarlisle** [comment]
@nelsonic goodparts >> `Doctype must be declared first.`

**Comment by nelsonic** [comment]
Darnit! ok. motion to _turn off_ that rule in `goodparts` ?
(_justification: both demos in `hapi-riot` pass W3C validation..._)

**Comment by jackcarlisle** [comment]
@nelsonic seconded! 🔨

**Comment by jackcarlisle** [comment]
@nelsonic actually I think it was `linter-htmlhint`...

**Comment by nelsonic** [comment]
Ah... thought so... ok ignore. 👍 
(_we will have 100% W3C-compliant code so it's not an issue..._)

File: .eslintignore [hunk 3]
```diff
@@ -0,0 +1 @@
+coverage
```
Comment (nelsonic) [inline .eslintignore:1]: pesky `.eslintignore` file with a single line... https://github.com/dwyl/goodparts/issues/255

**Comment by nelsonic** [comment]
@jackcarlisle looks good. merging. 👍

**Pull request merged**